src/cli.rs

```diff
@@ -1,16 +1,12 @@
 use image::codecs::png::{CompressionType, FilterType};
-use image::{ImageFormat, Rgba};
+use image::ImageFormat;
 use pconvert_rust::benchmark::Benchmark;
-use pconvert_rust::blending::{
-    blend_images, get_blending_algorithm, is_algorithm_multiplied, multiply_image, BlendAlgorithm,
-};
+use pconvert_rust::blending::BlendAlgorithm;
 use pconvert_rust::constants;
 use pconvert_rust::errors::PConvertError;
-use pconvert_rust::parallelism::{ResultMessage, ThreadPool};
-use pconvert_rust::utils::{read_png_from_file, write_png_parallel, write_png_to_file};
+use pconvert_rust::test::{apply_blue_filter, compose, compose_parallel, Background};
```

Comment:
let's move this stuff from the test namespace and to something better like `utils` or `compose`